ngrinder-controller/src/main/java/org/ngrinder/infra/config/Config.java

```diff
@@ -735,4 +735,25 @@ public long getInactiveClientTimeOut() {
 	public boolean isEnableStatistics() {
 		return getControllerProperties().getPropertyBoolean(PROP_CONTROLLER_ENABLE_STATISTICS);
 	}
+
+	/**
+	 * Get Csv Separator.
+	 * defaultValue is comma.
+	 *
+	 * @return String csvSeparatorValue
+	 */
+	public String getCsvSeparator() {
+		String csvSeparatorValue = getControllerProperties().getProperty(
+				PROP_CONTROLLER_CSV_SEPARATOR, "comma");

```

Comment:
You don't need to call getProperty(FIELD, defaultValue). just Call getProperty(FIELD)... and add the default value in controller in https://github.com/naver/ngrinder/blob/master/ngrinder-controller/src/main/resources/controller-properties.map

the format is like following..
```field_name,default_value,old_fieldname(can be ommitted)```

So it should be..
```controller.csv_separator,comma,```